templates/components/tools/editors.html.hbs

```diff
@@ -24,7 +24,7 @@
        class="button button-secondary">{{fluent "tools-editor-emacs"}}</a>
   </div>
   <div class="w-25-l w-50-m w-100 pa3">
-    <a href="https://geany.org/about/filetypes/"
-       class="button button-secondary">{{fluent "tools-editor-geany"}}</a>
+    <a href="https://rust-analyzer.github.io/manual.html#visual-studio-2022"
```

Comment:
https://github.com/rust-lang/rust-analyzer/pull/16361 adds an anchor without the year in it.